src/chain_service/faucet.ts

```diff
@@ -196,10 +196,20 @@ export default class Faucet {
   }
 
   private checkLackOfFunds(error: any) {
-    if (error.name === "RpcError") {
-      if (error.code === 1010) {
+    if (error.name === "RpcError" && error.message.includes("Inability to pay some fees")){
         this.registerLackOfFundsEvent();
-      }
+    }else{
+        console.log(
+          "\x1b[31m%s\x1b[0m",
```

Comment:
Shouldn't we leave it at 
```suggestion
          "\x1b[31m",
```

Reply:
As far as I understand this is supposed to color the output text red?

Reply:
I don't really know why I changed it. Rolling back.